projects/valtimo/dossier-management/src/lib/components/dossier-management-widget-tab/dossier-management-widget-tab.component.html

```diff
@@ -0,0 +1,42 @@
+<!--
+  ~ Copyright 2015-2024 Ritense BV, the Netherlands.
+  ~
+  ~ Licensed under EUPL, Version 1.2 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" basis,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<ng-container
+  *ngIf="{
+    currentWidgetTab: currentWidgetTab$ | async
+  } as obs"
+>
```

Comment:
If this is the only observable that's gonna go in here, then the obs object is not necessary. Please use currentWidgetTab$ as compactMode$ is used on line 34